spec/models/extraction_definition_spec.rb

```diff
@@ -204,8 +204,8 @@
         expect { subject.destroy }.to change(HarvestJob, :count).by(-1)
       end
 
-      it "destroys the harvest reports" do
-        expect { subject.destroy }.to change(HarvestReport, :count).by(-1)
+      it "does not destroy the harvest reports" do
```

Comment:
suggestion: came across this yesterday inspired by your TDD talk which may be of interest if not applicable here. https://rspec.info/features/3-12/rspec-core/subject/one-liner-syntax/